packages/core/src/llms/type.ts

```diff
@@ -54,7 +54,7 @@ export interface LLM<
   ): Promise<CompletionResponse>;
 }
 
-export type MessageType = "user" | "assistant" | "system" | "memory";
+export type MessageType = "user" | "assistant" | "system" | "memory" | "tool";
```

Comment:
@erik-balfe we currently don't use "tool", a tool message is one that has a `toolResult`, see the conversion for openai: https://github.com/run-llama/LlamaIndexTS/blob/9e648a0b4f670e131dc6f978872009f8939b2825/packages/providers/openai/src/llm.ts#L267

@himself65 i remember you added this

Reply:
Ok, just removed it now

Reply:
we put tool info into "user" and "assistant"